wrangler-transform/widgets/Wrangler-transform.json

```diff
@@ -73,6 +73,24 @@
         }
       ]
     },
+    {
+      "label": "RelationalDirectives",
```

Comment:
Do you want to keep these changes by feature flag? Otherwise we have to move this code to a branch and keep syncing the branch with Develop or other future changes.

Reply:
We will add feature flag. until then we won't merge to develop